app/components/search/SearchResults/SearchResults.jsx

```diff
@@ -44,7 +42,11 @@ const SearchResults = ({ searchResults, expandList, setExpandList }) => {
     return () => {
       document.body.classList.remove('searchResultsPage');
     };
-  }, []);
+  }, [googleMapObject, centerCoords]);
```

Comment:
I was looking at manually testing this, but it doesn't look like `setCenterCoords()` is ever called. I'm not going to remove this in this PR, but I think it's something we may want to do when we migrate this file to TypeScript, where it should hopefully be a lot more obvious when something is unused.

Reply:
I think it may be leftover from the multiple location work that was never fully implemented.  I will make a note to look into whether it can be of use to us.